external/src/main/java/com/redhat/cloud/policies/engine/process/Receiver.java

```diff
@@ -184,20 +191,25 @@ public Uni<Void> processAsync(Message<String> input) {
 
         event.setFacts(systemProfile);
 
-        try {
-            List<Event> eventList = new ArrayList<>(1);
-            eventList.add(event);
-            if (storeEvents) {
-                return alertsService.addEvents(eventList)
-                        .replaceWith(ack(input))
-                        .runSubscriptionOn(Infrastructure.getDefaultWorkerPool());
-            } else {
-                return alertsService.sendEvents(eventList)
-                        .replaceWith(ack(input))
-                        .runSubscriptionOn(Infrastructure.getDefaultWorkerPool());
+        if (lightweightEngineEnabled) {
+            lightweightEngine.process(event);
+            return Uni.createFrom().voidItem();
+        } else {
+            try {
+                List<Event> eventList = new ArrayList<>(1);
+                eventList.add(event);
+                if (storeEvents) {
```

Comment:
Isn't this something we could already remove? 
IIRC it's part of a previous migration to store in psql? or is something else?

Reply:
That branch of the `if` is already unused so it could be removed, yes.

However, I don't plan on removing anything yet. There will be a huge cleaning PR in the future, after we're done testing the new engine on prod.